### DISCUSSION TITLE
Week 6 Thinking Cap: Thick Mapping

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#51] [Assignments]
Topic: Thick mapping in the digital humanities

**Comment by lisa58** [comment]
https://github.com/lisa58/DH151/blob/main/Week6/W6%20Thinking%20Cap.md

**Comment by LynnT2** [comment]
https://github.com/LynnT2/DH151/blob/main/Week%206/W6%20Thinking%20Cap.md

**Comment by alicelu170** [comment]
https://github.com/alicelu170/DH151/blob/main/W6%20Thinking%20Cap.md

**Comment by jomh17** [comment]
https://github.com/jomh17/DH151/blob/main/Week%206/README.md

**Comment by MaryJaneHowland** [comment]
https://github.com/MaryJaneHowland/DH151/blob/main/Week%206/ThinkingCap.md

**Comment by justisg** [comment]
https://github.com/justisg/DH151/blob/main/Week6/ThinkingCap.md

**Comment by lisalou36** [comment]
**What are your thoughts on "thick mapping in the digital humanities," and how does it differ from "mapping?"**
  I was intrigued by the authors' discussion of "thick mapping" as an honest argument that history and data are never without bias and therefore can never be epitomize a singular truth. I think this is a key concept that is often glossed over both STEM, social science and humanities curriculum to push forward our own field's narrative as truth. Research can be thorough, methods incredibly precise, and the writing or modeling persuasive, but in the end the narrative still comes from a limited number of perspectives. It is impossible to completely document history, and in the world of archival science this is the ultimatum we must grapple with -- how do we decide what is worth saving? Thick mapping deconstructs the silos of STEM and the humanities/social science fields (an action I am a HUGE fan of) by literally illustrating how different fields can work together and what influence they can yield when they do so. Science and history are constantly developing and being discovered. This differs from the traditional understanding of mapping as a captured, singular moment of science, sociology, and history because it makes this singularity (one person's "truth") a multiplicity (a realistic blurring of the idea of truth). 

**How does the theory of "thick mapping" inform narratives that are influencing your life?**
  The theory of thick mapping motivates me to continue work that I want to do in information literacy and crosswalking STEM and the humanities (and all fields of study in between). I believe that every person and therefore society as a whole benefits from communicating with those who are different from you, and that includes fields of expertise and study. I think CS students could benefit from a class in Labor Studies, History students could learn from life sciences, Business majors from an Indigenous studies class. Our skills are extensible, interoperable, and worthy of critical analysis, and our lives are completely networked and entwined. Thick mapping eschews universalism in favor of polyvocality, so should education and the careers we pursue.

**Comment by kimjee8955** [comment]
https://github.com/kimjee8955/DH151/blob/main/Week6/ThinkingCap.md

**Comment by ryanorlando** [comment]
https://github.com/ryanorlando/Dh151/blob/main/Week6ThinkingCap.md

**Comment by michelle14lee** [comment]
https://github.com/michelle14lee/DH151/blob/main/Week6/Week6ThinkingCap.md

**Comment by jmtanenbaum** [comment]
https://github.com/jmtanenbaum/DH151/blob/main/thinkcap.md

**Comment by alekhyavittalam** [comment]
https://github.com/alekhyavittalam/DH151/blob/main/ThinkingCapWeek6.md

**Comment by adrianaromero819** [comment]
https://github.com/adrianaromero819/DH151/blob/main/Week6/Week%206%20%20Thinking%20Cap.md

**Comment by danielliuzy** [comment]
https://github.com/danielliu524/DH151/blob/main/Week6/W6%20Thinking%20Cap.md

**Comment by natgrace** [comment]
https://github.com/natgrace/DH151/blob/main/Week6/ThinkingCapW6.md

**Comment by anncong** [comment]
https://github.com/anncong/DH151/blob/main/Week6/W6_ThinkingCap.md

**Comment by ainsleighdouglas** [comment]
https://github.com/ainsleighdouglas/DH-151/blob/main/Week6/Week6_ThinkingCap.md

**Comment by hanarama** [comment]
https://github.com/hanarama/DH151/blob/main/Week%206/Thinking%20Cap.md

**Comment by zonaliao** [comment]
https://github.com/zonaliao/DH151/blob/main/W6%20Thinking%20Cap.md

**Comment by JuliaWood1** [comment]
https://github.com/JuliaWood1/DH151/blob/main/Week6/ThinkingCap.md

**Comment by kaicolorado** [comment]
https://github.com/kaicolorado/DH151/blob/main/Week%206/ThinkingCap.md

**Comment by yjwsb233** [comment]
https://github.com/yjwsb233/DH151/blob/main/Week6_ThinkingCap.md

**Comment by frida-barragan** [comment]
https://github.com/frida-barragan/DH151/blob/main/thinking%20cap_week6.md

**Comment by alexthegreat1** [comment]
https://github.com/alexthegreat1/DH151/blob/main/Week%206/W6%20Thinking%20Cap.md

**Comment by sebdills** [comment]
https://github.com/sebdills/DH151/blob/22d1f0433e25678c0890e34371edb2b67c7b5d72/thinkcap.md

**Comment by ivanma9** [comment]
https://github.com/ivanma9/DH151/blob/main/Week6/ThinkingCap:%20Thick%20Mapping.md

**Comment by hoz-map** [comment]
https://github.com/hoz-map/DH151/blob/main/Week6%20Thinking%20Cap.md

**Comment by kmcinerny** [comment]
https://github.com/kmcinerny/DH151/blob/main/Week6/thinkingcap.md

**Comment by ec028** [comment]
https://github.com/ec028/DH151/blob/main/Week6/thinkingcap.md

**Comment by derekwang99** [comment]
https://github.com/derekwang99/DH151/blob/main/Week6/ThinkingCap.md

**Comment by kaisanwatanabe** [comment]
https://github.com/kaisanwatanabe/DH151/blob/main/week%206/thinkingcap.md
